src/pages/location-message/location-message.scss

```diff
@@ -0,0 +1,14 @@
+.location-message {
+  .scroll-content {
+    margin-top: 44px;
+  }
+
+  sebm-google-map {
```

Comment:
I can't control the class, it's an internal of angular2-google-maps, and this is the usage of the module...